lifecycle/src/lifecycle_service_client.cpp

```diff
@@ -106,7 +106,7 @@ class LifecycleServiceClient : public rclcpp::Node
 
     // We send the service request for asking the current
     // state of the lc_talker node.
-    auto future_result = client_get_state_->async_send_request(request);
+    auto future_result = client_get_state_->async_send_request(request).future.share();
```

Comment:
This looks good, but don't we also need this fix on line 165 as well?

Reply:
oh you're right, I didn't think to check if there were other instances after checking that the demo worked. Fixed in edd8e2e2830aeb16a73ea39d33ba1a8054f47efb